src/controller.ts

```diff
@@ -9,7 +9,7 @@ import { Http2Client } from './Http2Client'
 
 const zeroconf = bonjour()
 
-function done() {
```

Comment:
I don't think the changes to this file should go in an unrelated unit test...but we're not in a place to be nitpicky so LOL go ahead

Reply:
Good point 😅, btw this change was needed to satisfy a linter warning.